components/KiboAppHeader.vue

```diff
@@ -106,15 +138,61 @@ export default defineComponent({
     const isAuthenticated = computed(() => {
       return userGetters.isLoggedInUser(user.value)
     })
-    const result = ref(null)
+    const term = ref("")
+    const isSearchOpen = ref(false)
+    const results = ref(null)
+    const searchBarRef = ref(null)
     const accountIcon = computed(() => {
       return isAuthenticated.value ? "fas" : "far"
     })
 
     const { toggleLoginModal, toggleStoreLocatorModal } = useUiState()
 
+    const closeSearch = () => {
+      if (!isSearchOpen.value) return
+
+      term.value = ""
+      isSearchOpen.value = false
+    }
+
+    const handleSearch = debounce(async (paramValue) => {
+      loading.value = true
```

Comment:
the composable should be updating this loading state

Reply:
Resolved with icky-305